backend/app/cmd/server.go

```diff
@@ -1215,10 +1215,15 @@ func (s *ServerCommand) getAuthenticator(ds *service.DataStore, avas avatar.Stor
 			if c.User == nil {
 				return c
 			}
-			c.User.SetAdmin(ds.IsAdmin(c.Audience, c.User.ID))
-			c.User.SetBoolAttr("blocked", ds.IsBlocked(c.Audience, c.User.ID))
+			if len(c.Audience) != 1 {
```

Comment:
I think this check needs some comments explaining what is going on here, specifically how 0 or >1 is causing this lack of update and why.